src/main/java/org/polyjdbc/core/dialect/MysqlDialectTypes.java

```diff
@@ -16,8 +16,19 @@
 package org.polyjdbc.core.dialect;
 
 /**
- *
  * @author Adam Dubiel
  */
 public class MysqlDialectTypes extends DefaultDialectTypes {
+
+    public String timestamp(int fractionalDigits) {
```

Comment:
these methods wont be used by poly, the idea is to override default types. I have fixed it here
https://github.com/polyjdbc/polyjdbc/pull/39/files
so TIMESTAMP(3) will be used always to persist datetime types